### ISSUE TITLE
Colliding default keyboard shortcuts

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#1658]
### Describe the problem

The default keyboard shortcuts come from two sources:

- The [**Eclipse Theia** IDE framework](https://theia-ide.org/) Arduino IDE 2.x is built on / the [**VS Code**](https://code.visualstudio.com/) application Theia is based on.
- Arduino IDE 1.x

🐛 Some keyboard shortcuts are used by both sources in the same context, but for different uses. In this case, the keyboard shortcut does not work for one of the uses, which will be confusing to the user.

- "**Toggle Output View**" / "**arduino-upload-sketch-using-programmer**"
- "**Format Document**" / "**Toggle Library Manager View**"
  - This only affects Linux; Windows and macOS use a different shortcut for "**Format Document**"

The list above is incomplete. A comprehensive review must be done of all keyboard shortcuts.

### To reproduce

1. Select **File > Advanced > Keyboard Shortcuts** from the Arduino IDE menus.
   A "**Keyboard Shortcuts**" view will open in the main panel.
1. In the "**Search keybindings**" field of the view, type `Toggle Output View`
1. Take note of the keyboard shortcut shown under the "**Keybinding**" column: <kbd>**Ctrl**</kbd>+<kbd>**Shift**</kbd>+<kbd>**U**</kbd> (<kbd>**Command**</kbd>+<kbd>**Shift**</kbd>+<kbd>**U**</kbd> for macOS users).
1. Press the <kbd>**Ctrl**</kbd>+<kbd>**Shift**</kbd>+<kbd>**U**</kbd> keyboard shortcut.

🐛 The IDE attempts an "**Upload Using Programmer**" operation instead of toggling the "**Output**" panel.

### Expected behavior

Default keyboard shortcuts do not collide with each other.

### Arduino IDE version

3a70547

### Operating system

All

### Operating system version

Any

### Additional context

Related

- https://github.com/arduino/arduino-ide/issues/945
- https://github.com/arduino/arduino-ide/issues/774

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My report contains all necessary details
